### ISSUE TITLE
Paths with urlescaped characters in their filenames break the tree-view

### ISSUE THREAD

**Issue by fake-name** [shirosaidev/diskover-web#17]
I have a directory with a bunch of files and folders where the names of the files/folders have urlescaped punctuation in their actual names. 

This appears to completely confuse at least the tree-view. In my case, there are folders which have urlescaped punctuation, and a folder named the exact same without the urlescaping (the root cause of this was a bug in a web spider on my end, but there shouldn't be a filename that can break the web-UI).

I also have some directories with newlines in the folder name. This is *technically* valid (at least on linux). I'm working to fix it, but it also appears to also confuse the web-up (the newlines appear to be just stripped). This issue might be just the HTML doing text-reflow.

Basically, I think there's a missed escape somewhere in the file processing.

**Comment by shirosaidev** [comment]
are the urlescaped punctuation and directory with newlines correct in the ES index when you look with Kibana or ES query?
